coverage-report/src/test/java/org/jooby/issues/PullRequest583.java

```diff
@@ -0,0 +1,37 @@
+package org.jooby.issues;
+
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import org.jooby.test.ServerFeature;
+import org.junit.*;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * @author Johannes Schneider (<a href="mailto:js@cedarsoft.com">js@cedarsoft.com</a>)
+ */
+public class PullRequest583 extends ServerFeature {
```

Comment:
This won't run from maven. Rename your test as `Issue583` or append `Test`, like `PullRequest583Test`